src/components/Accounts/ViewFeesFines/ViewFeesFines.js

```diff
@@ -171,6 +173,46 @@ class ViewFeesFines extends React.Component {
     />;
   }
 
+  showComments = (feeFine) => {
+    const feeFineType = feeFine.feeFineType ? feeFine.feeFineType : '';
+    const comments = _.get(this.props.resources, ['comments', 'records'], []);
+    const actions = _.orderBy(comments.filter(c => c.accountId === feeFine.id), ['dateAction'], ['asc']);
+    const myComments = actions.filter(a => a.comments).map(a => a.comments);
+    const commentsAmount = myComments.length;
+
+    return (
+      <div>
+        <Row>
+          <Col>{feeFineType}</Col>
+          {(commentsAmount > 0) ?
+            <Col style={{ marginLeft: '5px' }}>
```

Comment:
Could you please move style to css file?

Reply:
Done